cpp/models/ode_secir/parameters_io.h

```diff
@@ -166,17 +166,17 @@ IOResult<void> export_input_data_county_timeseries(std::vector<Model>& model, co
         for (size_t group = 0; group < ConfirmedCasesDataEntry::age_group_names.size(); group++) {
 
             t_InfectedNoSymptoms[county].push_back(
-                static_cast<int>(2 * (model[county].parameters.template get<IncubationTime>()[AgeGroup(group)] -
-                                      model[county].parameters.template get<SerialInterval>()[AgeGroup(group)])));
+                static_cast<int>(std::round(2 * (model[county].parameters.template get<IncubationTime>()[AgeGroup(group)] -
+                                      model[county].parameters.template get<SerialInterval>()[AgeGroup(group)]))));
             t_Exposed[county].push_back(
-                static_cast<int>(2 * model[county].parameters.template get<SerialInterval>()[AgeGroup(group)] -
-                                 model[county].parameters.template get<IncubationTime>()[AgeGroup(group)]));
+                static_cast<int>(std::round(2 * model[county].parameters.template get<SerialInterval>()[AgeGroup(group)] -
+                                 model[county].parameters.template get<IncubationTime>()[AgeGroup(group)])));
             t_InfectedSymptoms[county].push_back(
-                static_cast<int>(model[county].parameters.template get<TimeInfectedSymptoms>()[AgeGroup(group)]));
+                static_cast<int>(std::round(model[county].parameters.template get<TimeInfectedSymptoms>()[AgeGroup(group)])));
             t_InfectedSevere[county].push_back(
-                static_cast<int>(model[county].parameters.template get<TimeInfectedSevere>()[AgeGroup(group)]));
+                static_cast<int>(std::round(model[county].parameters.template get<TimeInfectedSevere>()[AgeGroup(group)])));
             t_InfectedCritical[county].push_back(
-                static_cast<int>(model[county].parameters.template get<TimeInfectedCritical>()[(AgeGroup)group]));
+                static_cast<int>(std::round(model[county].parameters.template get<TimeInfectedCritical>()[(AgeGroup)group])));
```

Comment:
There is some relation between the SECIR and SECIRVVS model so we should have the rounding also in the `set_confirmed_cases_data` equivalent in SECIR. Unfortunately, the function there is still named `set_rki_data`. Maybe you can correct for both. Name and rounding and check that it is consistent with both, the export function here and the function in secirvvs